example/webgpu.html

```diff
@@ -0,0 +1,206 @@
+        <button onclick="stopAnimation()">Stop</button>
+        <button onclick="createAnimation()">Create</button>
+        <button onclick="destroyAnimation()">Destroy</button>
+        <button onclick="save('gif')">save2gif</button>
+        <button onclick="save('tvg')">save2tvg</button>
+        <button onclick="save('png')">save2png</button>
+        <div>
+          <label for="reverseCheckbox">Reverse</label>
+          <input type="checkbox" id="reverseCheckbox" onchange="toggleReverse(event)">
+        </div>
+          
+        <div>
+          <label for="loopCheckbox">Loop</label>
+          <input type="checkbox" id="loopCheckbox" onchange="toggleLoop(event)" checked>
+        </div>
+        
+        <div>
+          <label for="speedRange">Speed</label>
+          <input type="range" id="speedRange" min="0.1" max="2" step="0.1" value="1" onchange="changeSpeed(this.value)">
+        </div>
+        
+        <div>
+          <label for="frameRange">Frame</label>
+          <input type="range" id="frameRange" min="1" max="100" step="1" value="1" onchange="seekFrame(this.value)">
+        </div>
+      </div>
+
+      <div class="lottie"></div>
+    </div>
+    <script>
+      let animation;
+      window.onload = function() {
+        createAnimation();
+      }
+
+      function createAnimation() {
+        if (animation) {
+          alert('Animation already exists');
+          return;
+        }
+
+        const lottie = document.querySelector('.lottie');
+        animation = document.createElement('lottie-player');
+
+        animation.autoPlay = true;
+        animation.loop = true;
+        animation.src = "https://raw.githubusercontent.com/thorvg/thorvg/main/examples/resources/lottie/masking.json";
+        animation.style.width = "3000px";
+        animation.style.height = "2000px";
+        animation.renderConfig = { renderer: 'wg' };
+
+        lottie.appendChild(animation);
+
+        animation = document.querySelector('lottie-player');
+        const frameRange = document.querySelector('#frameRange');
+
+        animation.addEventListener('frame', function(e) {
+          frameRange.value = e.detail.frame / animation.totalFrame * 100;
+        });
+      }
+
+      function pauseAnimation() {
+        if (!animation) {
+          alert('Animation is not created yet');
+          return;
+        }
+
+        animation.pause();
+      }
+
+      function stopAnimation() {
+        if (!animation) {
+          alert('Animation is not created yet');
+          return;
+        }
+
+        animation.stop();
+      }
+
+      function seekFrame(framePercentage) {
+        if (!animation) {
+          alert('Animation is not created yet');
+          return;
+        }
+
+        animation.seek(animation.totalFrames * framePercentage / 100);
+      }
+
+      function destroyAnimation() {
+        if (!animation) {
+          alert('No animation to destroy');
+          return;
+        }
+
+        animation.term();
```

Comment:
Need to confirm:
1. It really need to dispose the resources by users.
 - Could you please share the emscripten guide link?
2. If it does:
 - Should we merge animation.destroy() & animation.term() call??

Reply:
The function `destory` is expected to be called, here is [link](https://emscripten.org/docs/api_reference/module.html?highlight=destroy#Module.destroy) you can check.

For the term() and destory() functions, I made them in separated functinos so give users control the specific time when animation is no longer needed.

If it's burden to users, we can hide them and make them called when DOM or screen is disposed from system. Then users don't have to care of module destroy process.